### DISCUSSION TITLE
Coming from iamcco/markdown-preview

### DISCUSSION THREAD

**Discussion by Tweekism** [jannis-baum/Vivify#79] [General]
Hey mate,

I've come from the iamcco plugin as I see you have also, I was looking to revive the project but this might be better. It's a little hard to find a neovim markdown viewer and the top results on Google seems to be in various states of inactivity.

I haven't downloaded this yet, will do shortly. I don't have programming ability to undertake something like this on my own but hoping to find some way I can help out.

**Comment by jannis-baum** [comment]
Hello there!

Sounds great! Let me know in case you have questions, run into any issue, or have other interesting thoughts :) Since we don't have many users here yet any question and new perspective will help the project!

Aside from the things mentioned in the README we also have some other new features coming very soon (the open PRs) so stay tuned for the next release.

**Reply by Tweekism** [reply]
Yeah you are a little hard to find, searching neovim markdown on Google doesn't show you up. I like the idea that this can run standalone as well, but i never would have searched for it that way.

I'll let you know how I go after the install.

**Reply by Tweekism** [reply]
Ok well that was easy enough to install. Hmmm its getting pretty late here now in Australia, its 2 am. 

I'm going to go sleep on this now and will be back in the morning 😅

In the meantime, what are you guys on? Assuming MacOS and Arch?

**Reply by tuurep** [reply]
> Assuming MacOS and Arch?

Yep, this is true :D

How about you?

**Reply by Tweekism** [reply]
I'm on Fedora. Perhaps my first contribution can be working out how to package it for here.

I see you have one request for windows support, could be handy since all my team are on windows. Although they prefer sublime or vscode, which seem to have plenty of plugins.

**Reply by jannis-baum** [reply]
Nice! Yes, packaging is definitely appreciated! I still have to do that for macOS as well, have been pushing it off ^^

The windows support issue is because someone else asked for it. I think none of us will implement it, at least I won't since I don't have any windows machine. But in case you know anyone who would be willing to do it, feel free to point them in the direction, haha

**Reply by Tweekism** [reply]
I always keep a windows machine around, either for anticheat games at home or for windows tooling, sysadmin stuff at work.

Problem is once you add windows support you then also have to maintain it. I used to do programming for in-house windows desktop line of business applications in .net. Right now I could possibly manage it, but it could quickly grow out of hand and out of my ability.

Anyway this is probably a discussion for the other thread.

**Reply by tuurep** [reply]
I do have a win 10 machine so I can potentially test or something, but have no prior knowledge what's involved in windows-compatibility

**Reply by jannis-baum** [reply]
> Problem is once you add windows support you then also have to maintain it.

Yes, definitely! Will only make sense to add it if there's someone who is able and willing to take responsibility for that

**Reply by Tweekism** [reply]
> Nice! Yes, packaging is definitely appreciated! I still have to do that for macOS as well, have been pushing it off ^^

Ok, so packaging for Fedora in an undertaking on par with Frodo's quest to destroy the one ring. So I'm going to put that in my "ToDo" pile, while also keeping an eye out for some slightly lower hanging fruit.

**Reply by jannis-baum** [reply]
Hahaha, alright

**Comment by tuurep** [comment]
Cool :) I switched over from `markdown-preview.nvim` as well due to finding it cumbersome to make a small change to it:

https://github.com/tuurep/markdown-preview.nvim (my attempt to keep a fork in sync but now it's left behind)

For neovim I'm looking to fix a couple annoyances:

* https://github.com/jannis-baum/vivify.vim/issues/5
* https://github.com/jannis-baum/vivify.vim/issues/6

But other than that it's been great and I use it every day.

**Reply by Tweekism** [reply]
I use it at work, so I kinda need something, but I was having trouble getting it to build, had to set env vars to force legacy SSL for some reason and then missing deps app/out/ and then with my extremely limited node experience and everyone who have used the plugin seemingly already left, I wasn't really looking forward to it :P

**Reply by tuurep** [reply]
Oh yeah, looks like I was also hit with that legacy SSL problem based on the latest commit I made :smile: I didn't remember this.
